ubnt-edgerouter-mixin/dashboards/overview.jsonnet

```diff
@@ -0,0 +1,967 @@
+    tcpPassiveOpens: ratequery('tcpPassiveOpens', sharedMatcher),
+    tcpAttemptFails: ratequery('tcpAttemptFails', sharedMatcher),
+    tcpEstabResets: ratequery('tcpEstabResets', sharedMatcher),
+
+    tcpCurrEstab: ratequery('tcpCurrEstab', sharedMatcher),
+
+    tcpInSegs: ratequery('tcpInSegs', sharedMatcher),
+    tcpOutSegs: ratequery('tcpOutSegs', sharedMatcher),
+    tcpRetransSegs: ratequery('tcpRetransSegs', sharedMatcher),
+    tcpInErrs: ratequery('tcpInErrs', sharedMatcher),
+    tcpOutRsts: ratequery('tcpOutRsts', sharedMatcher),
+
+    udpInDatagrams: ratequery('udpInDatagrams', sharedMatcher),
+    udpOutDatagrams: ratequery('udpOutDatagrams', sharedMatcher),
+    udpInErrors: ratequery('udpInErrors', sharedMatcher),
+    udpNoPorts: ratequery('udpNoPorts', sharedMatcher),
+
+    icmpIpv4StatsInMsgs: ratequery('icmpStatsInMsgs', ipversionlabelmatcher('icmpStatsIPVersion', 1)),
+    icmpIpv4StatsOutMsgs: ratequery('icmpStatsOutMsgs', ipversionlabelmatcher('icmpStatsIPVersion', 1)),
+    icmpIpv4StatsInErrors: ratequery('icmpStatsInErrors', ipversionlabelmatcher('icmpStatsIPVersion', 1)),
+    icmpIpv4StatsOutErrors: ratequery('icmpStatsOutErrors', ipversionlabelmatcher('icmpStatsIPVersion', 1)),
+
+    icmpIpv6StatsInMsgs: ratequery('icmpStatsInMsgs', ipversionlabelmatcher('icmpStatsIPVersion', 2)),
+    icmpIpv6StatsOutMsgs: ratequery('icmpStatsOutMsgs', ipversionlabelmatcher('icmpStatsIPVersion', 2)),
+    icmpIpv6StatsInErrors: ratequery('icmpStatsInErrors', ipversionlabelmatcher('icmpStatsIPVersion', 2)),
+    icmpIpv6StatsOutErrors: ratequery('icmpStatsOutErrors', ipversionlabelmatcher('icmpStatsIPVersion', 2)),
+
+    icmpIpv4MsgStatsInPkts: ratequery('icmpMsgStatsInPkts', ipversionlabelmatcher('icmpMsgStatsIPVersion', 1)),
+    icmpIpv4MsgStatsOutPkts: ratequery('icmpMsgStatsOutPkts', ipversionlabelmatcher('icmpMsgStatsIPVersion', 1)),
+
+    icmpIpv6MsgStatsInPkts: ratequery('icmpMsgStatsInPkts', ipversionlabelmatcher('icmpMsgStatsIPVersion', 2)),
+    icmpIpv6MsgStatsOutPkts: ratequery('icmpMsgStatsOutPkts', ipversionlabelmatcher('icmpMsgStatsIPVersion', 2)),
+
+    ipRoutingDiscards: 'ipRoutingDiscards{' + sharedMatcher + '}',
+    inetCidrRouteNumber: 'inetCidrRouteNumber{' + sharedMatcher + '}',
+    ipForwardNumber: 'ipForwardNumber{' + sharedMatcher + '}',
+
+    snmpInPkts: ratequery('snmpInPkts', sharedMatcher),
+    snmpOutPkts: ratequery('snmpOutPkts', sharedMatcher),
+    snmpInGetNext: ratequery('snmpInGetNexts', sharedMatcher),
+    snmpInGetRequests: ratequery('snmpInGetRequests', sharedMatcher),
+    snmpInGetResponses: ratequery('snmpInGetResponses', sharedMatcher),
+    snmpOutGetNext: ratequery('snmpOutGetNexts', sharedMatcher),
+    snmpOutGetRequests: ratequery('snmpOutGetRequests', sharedMatcher),
+    snmpOutGetResponses: ratequery('snmpOutGetResponses', sharedMatcher),
+
+    snmpInTotalReqVars: ratequery('snmpInTotalReqVars', sharedMatcher),
+  },
+
+  panels:: {
+    integrationStatus:
```

Comment:
The fonts on these status panels look large as compared to other panels.

Reply:
Also once this [PR](https://github.com/grafana/cloud-onboarding/issues/2375) is merged, I think we should remove these status rows from jsonnet-lib mixins since they are cloud and integrations specific?

Reply:
Yep, good call. I removed these.